boards/posix/native_posix/doc/board.rst

```diff
@@ -620,3 +628,26 @@ can feed it the following set of commands through a pipe:
    echo -e \
    'select kernel\nuptime\n!wait 500\nuptime\n!wait 1000\nuptime\n!quit' \
    | zephyr/zephyr.exe
+
+Host based flash access
+***********************
+
+In case a flash device is present the file system partitions on the flash device
+can be exposed through the host file system by enabling
+:option:`CONFIG_NATIVE_POSIX_FUSE_FLASH_ACCESS`.
+
+By default the partitions are exposed through the directory *flash* in the
+current working directory. This directory can be changed via the option
+command line option *--flash-mount*.
```

Comment:
Some more documentation of this would not harm.. Right now the only documentation is this paragraph and the code (comment free ;) )